devel/README.md

```diff
@@ -0,0 +1,20 @@
+# Developer guide and tooling
+
+The Containerfile contained in this directory is intended to help with working on the multiple components that make up bootc-image-builder. Its purpose is to simplify building the bootc-image-builder container from locally checked out, development versions of the following projects:
+1. [osbuild/osbuild](https://github.com/osbuild/osbuild)
+2. [osbuild/images](https://github.com/osbuild/images)
+3. [osbuild/bootc-image-builder](https://github.com/osbuild/bootc-image-builder) (this repository)
+
+The Containerfile expects three extra build contexts, one for each component, pointing to the root of the source directory of each project. For example, given the following:
+```
+$ ls -1 ~/src
+images/
+osbuild/
+```
+the container can be built using:
+```
+$ podman build . --build-context=osbuild=$HOME/src/osbuild --build-context=images=$HOME/src/images --build-context=bib=.. -t bootc-image-builder:devel
```

Comment:
It feels more natural to me to run this from the root of bib, so instead of the extra build-context for bib use `-f ./devel/Containerfile` and drop the `--from=bib` from it.

Reply:
Hm yeah, I like that more too.  I'll update.

Reply:
Updated Containerfile and instructions.